types/google.script.client-side/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "dt-header": false
```

Comment:
Ahhhh.... why is this rule disabled? We use the header for powering publishing and the bot, so it being correct is rather important.